[2024-06-21T06:43:35.355+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-21T06:43:35.598+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: wikipedia_flow.transform_wikipedia_data manual__2024-06-21T06:43:14.712131+00:00 [queued]>
[2024-06-21T06:43:35.697+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: wikipedia_flow.transform_wikipedia_data manual__2024-06-21T06:43:14.712131+00:00 [queued]>
[2024-06-21T06:43:35.702+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-21T06:43:35.989+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_wikipedia_data> on 2024-06-21 06:43:14.712131+00:00
[2024-06-21T06:43:36.087+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'wikipedia_flow', 'transform_wikipedia_data', 'manual__2024-06-21T06:43:14.712131+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/wikipedia_flow.py', '--cfg-path', '/tmp/tmpeq7u_f2i']
[2024-06-21T06:43:36.131+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=23401) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-21T06:43:36.138+0000] {standard_task_runner.py:63} INFO - Started process 23412 to run task
[2024-06-21T06:43:36.150+0000] {standard_task_runner.py:91} INFO - Job 34: Subtask transform_wikipedia_data
[2024-06-21T06:43:36.645+0000] {task_command.py:426} INFO - Running <TaskInstance: wikipedia_flow.transform_wikipedia_data manual__2024-06-21T06:43:14.712131+00:00 [running]> on host dd278de8866b
[2024-06-21T06:43:37.513+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Ayush Goswami' AIRFLOW_CTX_DAG_ID='wikipedia_flow' AIRFLOW_CTX_TASK_ID='transform_wikipedia_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-21T06:43:14.712131+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-21T06:43:14.712131+00:00'
[2024-06-21T06:43:37.516+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-21T06:44:30.250+0000] {connectionpool.py:874} WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /search?q=Gazprom+Arena%2C%C2%A0Russia%0A&format=json&limit=1
[2024-06-21T06:47:11.332+0000] {connectionpool.py:874} WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /search?q=Stade+Mohammed+V+%2C%C2%A0Morocco&format=json&limit=1
[2024-06-21T06:48:17.961+0000] {connectionpool.py:874} WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /search?q=Stadium+Darul+Makmur%2C%C2%A0Malaysia&format=json&limit=1
[2024-06-21T06:48:23.803+0000] {connectionpool.py:874} WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /search?q=Taizhou+Sports+Centre+Stadium%2C%C2%A0China%0A&format=json&limit=1
[2024-06-21T06:48:49.839+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/pipeline/wikipedia_pipeline.py:76 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2024-06-21T06:48:50.347+0000] {python.py:237} INFO - Done. Returned value was: ok
[2024-06-21T06:48:50.348+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-21T06:48:50.856+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=wikipedia_flow, task_id=transform_wikipedia_data, run_id=manual__2024-06-21T06:43:14.712131+00:00, execution_date=20240621T064314, start_date=20240621T064335, end_date=20240621T064850
[2024-06-21T06:48:51.262+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-21T06:48:51.300+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-21T06:48:51.507+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
